### PR TITLE
fix(graphql): minor api prompt fixes

### PR DESCRIPTION
**Pull request by sundersc**
#### Description of changes
Minor GraphQL Api name and api key prompt changes.

#### Description of how you validated changes
- Manual test
- yarn test

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: packages/amplify-category-api/src/provider-utils/awscloudformation/service-walkthroughs/appSync-walkthrough.ts [hunk 1]
```diff
@@ -900,8 +907,8 @@ async function askOpenIDConnectQuestions(authSettings) {
   };
 }
 
-function validateDays(input) {
-  const isValid = /^\d+$/.test(input);
+async function validateDays(input) {
```
Comment (SwaySway) [inline packages/amplify-category-api/src/provider-utils/awscloudformation/service-walkthroughs/appSync-walkthrough.ts:910]: does this need to be async?

File: packages/amplify-category-api/src/provider-utils/awscloudformation/service-walkthroughs/appSync-walkthrough.ts [hunk 2]
```diff
@@ -832,7 +833,13 @@ export async function askApiKeyQuestions(authSettings = undefined) {
       default: defaultValues.apiKeyExpirationDays,
       validate: validateDays,
       // adding filter to ensure parsing input as int -> https://github.com/SBoudrias/Inquirer.js/issues/866
-      filter: value => (isNaN(parseInt(value, 10)) ? value : parseInt(value, 10)),
+      filter: value => {
+        let val = parseInt(value, 10);
```
Comment (cjihrig) [inline packages/amplify-category-api/src/provider-utils/awscloudformation/service-walkthroughs/appSync-walkthrough.ts:837]:
```suggestion
        const val = parseInt(value, 10);
```

**Review by cjihrig — APPROVED** [review]
LGTM with the comments addressed

File: packages/amplify-category-api/src/provider-utils/awscloudformation/service-walkthroughs/appSync-walkthrough.ts (hunk 1 above)
Comment (sundersc) [reply packages/amplify-category-api/src/provider-utils/awscloudformation/service-walkthroughs/appSync-walkthrough.ts:910]: Doesn't have to be async. But inquirer can handle validate method returning promise, so better to use it wherever possible.

File: packages/amplify-category-api/src/provider-utils/awscloudformation/service-walkthroughs/appSync-walkthrough.ts (hunk 2 above)
Comment (sundersc) [reply packages/amplify-category-api/src/provider-utils/awscloudformation/service-walkthroughs/appSync-walkthrough.ts]: updated.

**Review by SwaySway — APPROVED** [review]

**Pull request merged**